apps/metadata-editor/src/app/dashboard/search-filters-summary/search-filters-summary.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, Input } from '@angular/core'
+import { CommonModule } from '@angular/common'
+import { SearchFacade, SearchService } from '@geonetwork-ui/feature/search'
+import { map } from 'rxjs'
+import { SearchFiltersSummaryItemComponent } from '../search-filters-summary-item/search-filters-summary-item.component'
+import { TranslateModule } from '@ngx-translate/core'
+
+@Component({
+  selector: 'md-editor-search-filters-summary',
```

Comment:
this can maybe go in the feature-search lib ?

Reply:
Also, good point! I've moved it. I would've liked to separate logic and UI more to make it more reusable, but could not get it done. Maybe something for the future.